python/ray/train/backends/backend.py

```diff
@@ -137,6 +145,57 @@ def start(self,
             self._increment_failures()
             self._restart()
 
+    def _create_placement_group(self):
+        """Creates a placement group if it does not exist.
+
+        If a placement group is already detected (Tune) this will be a no-op.
+
+        By default the placement group will be created with PACK strategy.
+        This is optimized for colocating GPUs on a minimal number of nodes.
+        This behavior can be overridden to use the SPREAD strategy by defining
+        ``TRAIN_ENABLE_WORKER_SPREAD_ENV``
+
+        If a placement group is created it will be stored as
+        self._placement_group.
+        """
+        current_placement_group = get_current_placement_group()
+        should_capture_child_tasks_in_placement_group = \
```

Comment:
Man I wish there was a better way to handle this. Do we have a shared doc with Tune on asks for placement group?

Reply:
Created https://github.com/ray-project/ray/issues/20196 to track this!